latexify/core.py

```diff
@@ -32,44 +32,50 @@
 }
 
 class LatexifyVisitor(ast.NodeVisitor):
+  """Latexify AST visitor."""
 
   def __init__(self, math_symbol):
     self.math_symbol = math_symbol
-    super(ast.NodeVisitor).__init__()
+    super().__init__()
 
   def _parse_math_symbols(self, val: str) -> str:
     if not self.math_symbol:
       return val
     if val in _MATH_SYMBOLS:
       return '{\\' + val + '}'
-    else:
-      return val
+    return val
 
   def generic_visit(self, node):
     return str(node)
 
+  # pylint: disable=invalid-name
```

Comment:
If this is intended to allow visitor names, it would be better to add inline disabling for each `def`.

Reply:
Done